src/formatDate.js

```diff
@@ -1,14 +1,54 @@
 'use strict';
 
-/**
- * @param {string} date
- * @param {string[]} fromFormat
- * @param {string[]} toFormat
- *
- * @returns {string}
- */
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const prevSplitter = fromFormat[3];
+  const newSplitter = toFormat[3];
+  const rawDate = date.split(prevSplitter);
+  const newRawDate = [];
+
+  function setTheDate(dateType, nextDateType = dateType) {
+    newRawDate[toFormat.indexOf(nextDateType)] =
+      rawDate[fromFormat.indexOf(dateType)];
+  }
+
+  setTheDate('DD');
+  setTheDate('MM');
```

Comment:
The function calls for setting the date types 'DD' and 'MM' are correct, but ensure that the order of these calls matches the order in the 'toFormat' array.